steps/circuit.py

```diff
@@ -16,19 +16,13 @@
     save_circuit_set,
 )
 from zquantum.core.testing import create_random_circuit as _create_random_circuit
-from zquantum.core.utils import create_object
+from zquantum.core.utils import create_object, load_from_specs
 import json
 from typing import Union, List, Optional, Dict
 
 Specs = Union[str, Dict]
```

Comment:
@mstechly let's move the definition of the `Specs` type as well. Does that sound reasonable?

Reply:
Actually in another PR (https://github.com/zapatacomputing/z-quantum-core/pull/213) we're adding a `typing` module, I think this will be a more appropriate place to do so.

Reply:
Okay, fair enough. Just going to reference this thread on that PR so that we do this before merging PR #213